### ISSUE TITLE
Upgraded to 0.2.4 and gutenberg_works cannot find 140

### ISSUE THREAD

**Issue by rressler** [ropensci/gutenbergr#47]
I was having students use gutenburg_works to find the id for "the Jungle" by Upton Sinclair. My code worked fine and their code did as well. However, when we upgraded from 0.2.3 to 0.2.4 the book no longer appears in the list of works.

gutenberg_works() %>%
  filter(gutenberg_id == 140) # no work found with ID 140

Empty tibble

I can download it with gutenberg_downloads fine. Is there a reason that it no longer shows in the list of works but can still be downloaded?

https://www.gutenberg.org/ebooks/140

Thanks,
Richard

**Comment by jonthegeek** [comment]
I can see that it's being filtered out via the `only_text` filter (it has `FALSE` in that column). I'm digging in to see *why* that value is `FALSE` (whether it's an issue with our processor or an issue with the metadata). More soon, hopefully! (it takes a while to re-process the data to find where it fails)

**Comment by jonthegeek** [comment]
There was an inconsistency in the text filenames (we use the list of filenames to determine whether download will work). I'm testing out a fix right now to make sure it does what I think it will do!

**Comment by Myfanwy** [comment]
The fix solves the issue on my machine as well - although I feel it's worth mentioning that in the ~10 days since the PR was opened, they've added 16 new authors and 82 new works (!)

**Comment by rressler** [comment]
Thanks for the quick fix. I can’t see a daily build to download so here is the other test case I had. Several of the books in the download chunk no longer appeared (in the pre-fixed) list of works.
Thanks!
Richard


```{r}
gutenberg_authors[(str_detect(gutenberg_authors$author, "Twain")), ]
gutenberg_works(gutenberg_author_id == 53) |>
  filter(!str_detect(title, "(Part )|(Volume )|Chapter")) |>
  filter(str_detect(title, "(Sawyer)|(Connecticut)|(Mississippi)|(Prince)|(Tramp)|(Innocents)")) |>
           arrange(gutenberg_id)
```

- Download the works and show the distinct titles in the data frame.

```{r}
#| include: true
#| message: false
twain <- gutenberg_download(c(74, 86, 119, 245, 1837, 3176),
                            meta_fields = "title")
twain |> select(title) |>
  distinct()
```
From: Jon Harmon ***@***.***>
Date: Friday, December 8, 2023 at 11:05 PM
To: ropensci/gutenbergr ***@***.***>
Cc: Richard Ressler ***@***.***>, Author ***@***.***>
Subject: Re: [ropensci/gutenbergr] Upgraded to 0.2.4 and gutenberg_works cannot find 140 (Issue #47)

External Email: Use caution with links and attachments.

Closed #47<https://urldefense.com/v3/__https:/github.com/ropensci/gutenbergr/issues/47__;!!IaT_gp1N!3iu2_0Rbu3BrmF-2WvTAy1ZB_Vg0gnLhIe55Nh5jGLPOFSFVW1hbRKOKtI2vwm7XvcJcNKWECS4WCH9a8YeE23CUtiU$> as completed via #51<https://urldefense.com/v3/__https:/github.com/ropensci/gutenbergr/pull/51__;!!IaT_gp1N!3iu2_0Rbu3BrmF-2WvTAy1ZB_Vg0gnLhIe55Nh5jGLPOFSFVW1hbRKOKtI2vwm7XvcJcNKWECS4WCH9a8YeE9dU45uw$>.
